Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM gcr.io/google.com/cloudsdktool/cloud-sdk:328.0.0
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:395.0.0
 
 RUN apt-get install -qqy unzip
 
-ENV GOOGLE_APP_ENGINE_SDK_VERSION=1.9.70
```

Comment:
I believe this is the latest one available as you can see here:
https://console.cloud.google.com/storage/browser/appengine-sdks/featured;tab=objects?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%257B_22k_22_3A_22_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_22go_appengine_sdk_linux_amd64-1.9._5C_22_22%257D%255D%22))&prefix=go_appengine_sdk_linux_amd64-1.9.&forceOnObjectsSortingFiltering=false